src/install/python/python-frameworks/python-agent-cherrypy.mdx

```diff
@@ -0,0 +1,37 @@
+---
+componentType: appInfo
+optionType: framework
+headingText: Review start-up steps using CherryPy
+---
+
+You need to integrate the Python agent with your application so that your app's major functions and web requests are received by the agent. To integrate the Python agent with your app, you can run the newrelic-admin script in front of your usual app startup command.
+
+
+### Admin script integration method [#integrated-server]
```

Comment:
Remove this section entirely since it's the same as the server instructions.

Reply:
@hmstepanek I removed the section "Admin script integration method" and made a few minor edits.